src/js/pages/services/DeploymentsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import AlertPanel from '../../components/AlertPanel';
+
+var DeploymentsTab = React.createClass({
```

Comment:
It would be really nice to know about those sorts of conventions before tripping over them at review-time, so I was thinking about some new documentation to cover it.